### PR TITLE
SMHE-1322: Add DecoupleMbusDeviceAdministrative function

### PR DESCRIPTION
**Pull request by abolder**
Adds a new operation DecoupleMbusDeviceAdministrative for decoupling a g-Meter when an e-meter is not available and administrative decoupling should occur.

### CODE REVIEW

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature [hunk 1]
```diff
@@ -0,0 +1,90 @@
+@SmartMetering @Platform @NightlyBuildOnly @SMHE1322
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature]: Custom annotatie mag weg

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      | MbusPrimaryAddress             |                 9 |
+      | MbusManufacturerIdentification | LGB               |
+      | MbusVersion                    |                66 |
+      | MbusDeviceTypeIdentification   |                 3 |
+    And device simulation of "TEST1024000000001" with M-Bus client version 0 values for channel 1
+      | MbusPrimaryAddress             | 0 |
+      | MbusIdentificationNumber       | 0 |
+      | MbusManufacturerIdentification | 0 |
+      | MbusVersion                    | 0 |
+      | MbusDeviceTypeIdentification   | 0 |
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature]: Please add some values, to verify values are not changed or set to 0

Comment (kroesctrl) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature]: | MbusPrimaryAddress             | 3        |
      | MbusIdentificationNumber       | 12056731 |
      | MbusManufacturerIdentification | LGB      |
      | MbusVersion                    | 66       |
      | MbusDeviceTypeIdentification   | 3        |
      
And verify these values do no changes after response

**Review by kroesctrl — CHANGES_REQUESTED** [review]

**Comment by abolder** [comment]
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/41/

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature (hunk 1 above)
Comment (abolder) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature]: Check

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature (hunk 2 above)
Comment (abolder) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature]: Check

**Review by kroesctrl — APPROVED** [review]

**Comment by abolder** [comment]
Pre nightly:  https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/42/

**Comment by abolder** [comment]
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/44/
Geeft een KEY_NOT_PRESENT error op Lls1Connector.

**Comment by abolder** [comment]
Geslaagd https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/45/

**Pull request merged**